aten/src/ATen/native/native_functions.yaml

```diff
@@ -362,12 +362,11 @@
 
 - func: bitwise_not_(Tensor(a!) self) -> Tensor(a!)
   variants: method
-  dispatch:
-    CPU: _bitwise_not__cpu
 
 - func: bitwise_not(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   dispatch:
-    CPU: _bitwise_not_out_cpu
+    CPU: bitwise_not_out
```

Comment:
No need to list them if the dispatch to the same function.

Reply:
There are other places like this, e.g. https://github.com/pytorch/pytorch/blob/d9e15bccb0a127314850567aaf0fea0a6364c3eb/aten/src/ATen/native/native_functions.yaml#L2753-L2754 where CPU and CUDA dispatches go to the same functions - I assumed this is so that sparse tensors error out with a nice message if they call this function? If that's the case, makes sense to leave it here too.